libarchive/archive_disk_acl_darwin.c

```diff
@@ -76,8 +76,8 @@ static const acl_perm_map_t acl_nfs4_perm_map[] = {
 #endif
 };
 
-static const int acl_nfs4_perm_map_size =
-    (int)(sizeof(acl_nfs4_perm_map)/sizeof(acl_nfs4_perm_map[0]));
+static const size_t acl_nfs4_perm_map_size =
+	(sizeof(acl_nfs4_perm_map) / sizeof(acl_nfs4_perm_map[0]));
```

Comment:
This is an unnecessary style change:  I generally prefer to use `int` unless there's a need to do otherwise.  In this case, the map is a fixed list (just above) that has less than 20 entries.  So `int` works perfectly fine and reduces the need for extra casting below.

Reply:
No casting because i becomes size_t anyway.